packages/console/src/components/ItemPreview/UserPreview.tsx

```diff
@@ -9,17 +9,26 @@ import UserAvatar from '../UserAvatar';
 import ItemPreview from '.';
 
 type Props = {
-  user: User;
+  user: Pick<User, 'id' | 'avatar' | 'name' | 'primaryEmail' | 'primaryPhone' | 'username'> &
+    Partial<Pick<User, 'isSuspended'>>;
```

Comment:
Better leave some comments?  Can't follow the changes here.

Reply:
Sometimes the server returns partial user information instead of the full `User` schema type. Here only a subset of `User` type is needed to render the `UserPreview` component.

Reply:
Code updated